java/code/src/com/redhat/rhn/domain/server/test/ServerFactoryTest.java

```diff
@@ -254,7 +273,7 @@ public void testServerGroups() throws Exception {
 
         Collection servers = new ArrayList();
         servers.add(server);
-        ServerGroupManager manager = ServerGroupManager.getInstance();
+        ServerGroupManager manager = GlobalInstanceHolder.SERVER_GROUP_MANAGER;
```

Comment:
I was wondering here: even if you keep an instance of `ServerGroupManager` above (`private serverGroupManager`), why are you using `GlobalInstanceHolder.SERVER_GROUP_MANAGER` and assign it to another variable? Couldn't you just use the above instance?

Reply:
Yeah you are right. `ServerGroupManager` in tests is mostly just automatic refactoring and i have not gone over all cases and checked if they could be simplified more. I will check those now since it came up multiple times in this review already.